pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -331,3 +335,15 @@ type UProbeSpec struct {
 	// Selectors to apply before producing trace output. Selectors are ORed.
 	Selectors []KProbeSelector `json:"selectors"`
 }
+
+type ListSpec struct {
+	// +kubebuilder:validation:Optional
+	// Name of the list
+	Name string `json:"name"`
+	// +kubebuilder:validation:Optional
+	// Values of the list
+	Values []string `json:"values"`
+	// +kubebuilder:validation:Optional
+	// Indicates whether the symbols in list are syscalls.
+	Syscall bool `json:"syscall"`
```

Comment:
Should we have an enum type here so that it can be extended to other types if needed?

Reply:
hum, so at the moment we have the verification only for syscalls based on this flag,
do you have some other types in mind?

Reply:
IP addresses as well.

Reply:
ah right, so have it as general purpose list.. I was stuck with functions ;-) sounds good, will change that